packages/core/src/components/forms/_control-group.scss

```diff
@@ -215,6 +215,35 @@ Styleguide components.forms.control-group
   }
 
   /*
+  Responsive control groups
+
+  Add the class `pt-fill` to a control group to make all controls expand equally to fill the
+  available space. Then add the class `pt-fixed` to individual controls to revert them to their
+  original default sizes.
+
+  You can adjust the specific size of a control with the `flex-basis` CSS property.
+
+  Markup:
+  <div class="pt-control-group pt-fill">
+    <div class="pt-input-group">
+      <span class="pt-icon pt-icon-people"></span>
+      <input type="text" class="pt-input" placeholder="Find collaborators..." />
+    </div>
+    <button class="pt-button pt-intent-primary pt-fixed">Add</button>
+  </div>
+
+  Styleguide components.forms.control-group.fill
+  */
+
+  &.pt-fill {
+    display: flex;
```

Comment:
@leebyp Actually, this is exactly what I needed to match this API exactly with what button groups do. I'm gonna add it to this PR 👍  To summarize, we have two use cases in both button groups and control groups:
1. `.pt-fill` on the parent. Everything fills, but you can restrict certain elements by adding `.pt-fixed` to it.
1. `.pt-fill` on children elements directly, some or all.

How's that?